[debug] [2020-04-17T15:15:36.124Z] ----------------------------------------------------------------------
[debug] [2020-04-17T15:15:36.125Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\BluRRayS\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init
[debug] [2020-04-17T15:15:36.125Z] CLI Version:   8.1.0
[debug] [2020-04-17T15:15:36.125Z] Platform:      win32
[debug] [2020-04-17T15:15:36.125Z] Node Version:  v12.16.1
[debug] [2020-04-17T15:15:36.126Z] Time:          Fri Apr 17 2020 17:15:36 GMT+0200 (GMT+02:00)
[debug] [2020-04-17T15:15:36.126Z] ----------------------------------------------------------------------
[debug] [2020-04-17T15:15:36.126Z] 
[debug] [2020-04-17T15:15:36.133Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-17T15:15:36.152Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\BluRRayS\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async GoogleAuth.getClient (C:\Users\BluRRayS\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (C:\Users\BluRRayS\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
